.gitignore

```diff
@@ -0,0 +1,103 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
That works for the project level .gitignore file but I think more should be added to ignore the user specific .idea files you have commented below. I will update accordingly.